starter_templates/cpp/vcpkg.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": [
+    "fmt"
```

Comment:
Will remove before deploy.

Reply:
Actually, maybe we should include a dependency that is likely to be used in this challenge (like we do with the sha1 crate for Rust in bittorrent etc.)